libs/SalesforceSDK/src/com/salesforce/androidsdk/ui/OAuthWebviewHelper.java

```diff
@@ -462,13 +462,17 @@ protected class AuthWebViewClient extends WebViewClient {
 
         @Override
 		public void onPageFinished(WebView view, String url) {
-            // Remove the Biometric Login button from the Connected App allow denny screen.
+            // Remove the native login buttons (biometric, IDP) once on the allow/deny screen
```

Comment:
Hiding "Biometric Login" and "Login with IDP" buttons if user chose to do a "manual" login.